packages/overlay/src/spectrum-config-utils.js

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+// This transformation loosens up placement selector matching for positioned
+// overlays (e.g. [position="top"] => [position*="top"])
+module.exports.wildcardPlacementSelectorTransform = (selector) => {
```

Comment:
It would be nice to see a Spectrum CSS issue with the altering of this selector at the source brought up. Or we could include it in our sync next week, just the same.

Reply:
the `placement="top"` is already a translation from what is in Spectrum CSS. They just have: https://github.com/adobe/spectrum-css/blob/master/components/tooltip/index.css#L107-L115

Maybe we just need to add an option to the css processing for fuzzy matches?

Reply:
Oh right! That's what I really meant then. Thanks for clarifying that.

Reply:
I took another pass at this and am happier. I added a `wildcard` option for enums in the spectrum processing and eliminated the custom replacement function.

Reply:
👏